libexec/cli/instance.list.exec

```diff
@@ -33,6 +33,7 @@ if ! USERID=`id -ru`; then
     exit 255
 fi
 
+JSONFMT="false"
```

Comment:
Instead of doing this we tend to do something like: 
```
"${JSONFMT:-}"
```
That way a user can set the env var and in their environment and control the format that way rather than passing the argument.